includes/class-give-cache.php

```diff
@@ -114,10 +114,14 @@ public static function prevent_caching() {
 			)
 		);
 
-		if (
-			is_page( $page_ids )
-			|| is_singular( 'give_forms' )
-		) {
+		/**
+		 * Use this filter to prevent
+		 *
+		 * @since 2.6.3
+		 */
+		$canCache = apply_filters( 'give_can_cache_page', is_page( $page_ids ) );
```

Comment:
@ravinderk shouldn't this variable be called `$dontCache` in naming because you have this _preventing_ caching when it passes the condition and within that condition it sets the nocache headers?

Reply:
@DevinWalker yes, I am preventing caching here with `give_can_cache_page` filter. If review code then you will find that for `success_page` admin settings etc. `$canCache` will be true that means we can cache it and vise versa. I am using the affirmative name here. 
In the case of negative name ( `$dontCache`), we have to return `false` to the cache page.

Let me know if you want me to rename it.